fblldbviewhelpers.py

```diff
@@ -87,3 +87,6 @@ def upwardsRecursiveDescription(view, maxDepth=0):
     currentPrefix += "   | "
 
   return builder
+
+def slowAnimation(speed=1):
+  lldb.debugger.HandleCommand('expr -- for (UIWindow *$w in (NSArray *)[[UIApplication sharedApplication] windows]) { [[$w layer] setSpeed:(CGFloat)%s]; }' % (speed))
```

Comment:
This is awesome!

For what it's worth, another way that might have worked is to use `NSArray`'s custom implementation of `-setValue:forKey:`:

```objc
[[[UIApplication sharedApplication] windows] setValue:@(speed) forKey:@"speed"]
```